packages/material-ui/src/FormLabel/FormLabel.js

```diff
@@ -58,13 +66,14 @@ const FormLabel = React.forwardRef(function FormLabel(props, ref) {
   const fcs = formControlState({
     props,
     muiFormControl,
-    states: ['required', 'focused', 'disabled', 'error', 'filled'],
+    states: ['color', 'required', 'focused', 'disabled', 'error', 'filled'],
   });
 
   return (
     <Component
       className={clsx(
         classes.root,
+        classes[`color${capitalize(fcs.color || 'primary')}`],
```

Comment:
Where is the colorPrimary class defined?

Reply:
It's not